### PR TITLE
Streaming patch

### PR DESCRIPTION
**Pull request by JupiterJones**
When attempting to use the Streaming server in pharo (and GemStone) there were a couple of minor issues with:

GRCountingStream>>javascriptOn: aStream (was missing)
WAResponse>>writeStatusOn: aStream (was adding characters which caused an error with WAComboResponse)

Also added is a new class in Seaside-GemStone-Adaptors-Zinc to support streaming in GemStone.

Annoyingly there are a bunch of other changes grouped in here by exporting from GemStone which removes extra spaces and end of line characters.

### CODE REVIEW

File: repository/Seaside-Core.package/WAResponse.class/instance/writeStatusOn..st [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
\ No newline at end of file
+writeStatusOn: aStream
+  aStream nextPutAll: self httpVersion.
+  aStream
+    nextPutAll: $  greaseString;
+    nextPutAll: self status greaseString.
+  self message isNil
+    ifFalse: [ 
+      aStream
+        nextPutAll: $  greaseString;
```
Comment (theseion) [inline repository/Seaside-Core.package/WAResponse.class/instance/writeStatusOn..st]: Is this really necessary? Not that it's a huge deal, but the iteration overhead for single character + an additional method activation seem a bit ugly.

File: repository/Javascript-Core.package/GRCountingStream.extension/instance/javascript..st [hunk 2]
```diff
@@ -0,0 +1,3 @@
+*javascript-core
+javascript: anObject
+  anObject javascriptOn: self
```
Comment (theseion) [inline repository/Javascript-Core.package/GRCountingStream.extension/instance/javascript..st]: This is necessary as GRCountingStream does not extend Stream (which would have the method).

File: repository/Seaside-Core.package/WAResponse.class/instance/writeStatusOn..st (hunk 1 above)
Comment (marschall) [inline repository/Seaside-Core.package/WAResponse.class/instance/writeStatusOn..st]: I would prefer to leave it at #nextPut: as well

**Comment by marschall** [comment]
@JupiterJones the PR seems to reformat a lot of methods, can you undo this as this makes in unnecessarily hard to review?

**Comment by marschall** [comment]
- `GRCountingStream >> #javascriptOn:` was fixed by #938
- `WAResponse >> #writeStatusOn:`, "caused an error with WAComboResponse" can you be more specific? If `#nextPutAll:` with a collection of characters works then `#nextPut:` with a single character should work as well.

**Comment by theseion** [comment]
I'll attempt to extract the actual and valid changes and update the PR.

**Comment by theseion** [comment]
This PR has been superseded by #1075.
- The change to `#baselinefileSystem:` was already fixed in 8e8299409220c8f645da57f79beff9d97d1298c1
- The missing extension for `#javascript:` was already added in b9ce986589d7e088b945189d49f753da4bc8a2a0
- I've ignored the change to `#writeStatusOn:` as we don't have a clear error report

**Pull request closed without merging**